project-fortis-pipeline/docs/production-setup.md

```diff
@@ -1,14 +1,21 @@
-# Fortis production setup
+# Fortis production 
 
 ## Prerequisites
 
 ### Azure subscription
 
-First and foremost, you'll need an Azure subscription. You can create one for
-free [here](https://azure.microsoft.com/en-us/free/).
+First and foremost, you'll need an Azure subscription. 
+
+Azure is a comprehensive set of cloud services that developers and IT professionals use to build, deploy, and manage applications through our global network of datacenters. Integrated tools, DevOps, and a marketplace support you in efficiently building anything from simple mobile apps to internet-scale solutions.
+
+A full list of products and services can be found [here](https://azure.microsoft.com/en-us/services/).
```

Comment:
Please add a listing of the Azure resources that Fortis specifically uses for context. You can refer to the architecture diagram at https://user-images.githubusercontent.com/1086421/36809325-9b4f0542-1c95-11e8-9817-6af1939e4a1b.png to compile this list.